### PR TITLE
feat: docs

### PR DESCRIPTION
**Pull request by stephancill**
Adds initial vocs docs

- [x] Script to update spec
- [x] Best practicies
- [x] Live examples

### CODE REVIEW

File: docs/pages/best-practices.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Open Frames - Best Practices"
+description: "An extension of Open Graph which makes link previews interactive."
+---
+
+# Best Practices
+
+Open Frames is a powerful tool for creating interactive link previews. Below are some best practices to keep in mind when building Open Frames frames.
```
Comment (davidfurlong) [inline docs/pages/best-practices.mdx]: `Open Frames.`

File: docs/pages/best-practices.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Performance
+
+For the best UX, frames should be fast and snappy. Clients are encouraged to enforce a 5 second timeout, which is something to be aware of as a frame developer.
```
Comment (davidfurlong) [inline docs/pages/best-practices.mdx]: `frames developer.`

File: docs/pages/ecosystem/farcaster.mdx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Open Frames - Farcaster"
+description: "An extension of Open Graph which makes link previews interactive."
+---
+
+# Farcaster
+
+Open Frames is not supported by Farcaster, but most of the Open Frames properties are directly compatible with Farcaster Frames.
```
Comment (davidfurlong) [inline docs/pages/ecosystem/farcaster.mdx]: would phrase this differently - as there are Farcaster apps that support Open Frames in addition to Farcaster Frames. `Open Frames isn't officially supported by Farcaster, but most of the Open Frames properties are directly compatible with Farcaster Frames, and libraries for building Frames make it easy to support both conventions`

File: docs/pages/ecosystem/lens.mdx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Open Frames - Lens"
+description: "An extension of Open Graph which makes link previews interactive."
+---
+
+# Lens
+
+Open Frames is supported by Lens Protocol.
```
Comment (davidfurlong) [inline docs/pages/ecosystem/lens.mdx]: not sure which stage the LIP is at, but this text should probably reflect that

File: docs/pages/quickstart/onchainkit.mdx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Open Frames - OnchainKit"
+description: "An extension of Open Graph which makes link previews interactive."
+---
+
+# OnchainKit
+
+[OnchainKit](https://onchainkit.xyz/) is a library of React components and TypeScript utilities for building onchain applications with support for Open Frames frames.
```
Comment (davidfurlong) [inline docs/pages/quickstart/onchainkit.mdx]: `Open Frames.`

**Review by davidfurlong — CHANGES_REQUESTED** [review]
Looking good - almost there

File: vocs.config.ts [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+      ],
+    },
+  ],
+  socials: [
+    {
+      icon: "github",
+      link: "https://github.com/open-frames/standard",
+    },
+    {
+      icon: "warpcast",
```
Comment (davidfurlong) [inline vocs.config.ts]: maybe should be a farcaster logo

File: vocs.config.ts [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+      link: "https://warpcast.com/~/channel/open-frames",
+    },
+    {
+      icon: "x",
+      link: "https://x.com/openframesxyz",
+    },
```
Comment (davidfurlong) [inline vocs.config.ts]: We have the Lens @openframes handle and can be reached on XMTP via openframes.eth - perhaps we can include those for now as well

**Review by davidfurlong — CHANGES_REQUESTED** [review]
Would make the communit

File: vocs.config.ts (hunk 6 above)
Comment (stephancill) [reply vocs.config.ts]: icon support is quite limited 
Screenshot 2024-04-16 at 23 40 41

Comment (stephancill) [reply vocs.config.ts]: Screenshot 2024-04-16 at 23 51 33

i've converted the social icons to links. lmk what you think @davidfurlong

**Review by davidfurlong — APPROVED** [review]

**Pull request merged**